ZhuoHeiChaAPI/Services/ClientNotificationService.cs

```diff
@@ -79,20 +82,26 @@ public async Task SendInitalGamePackage(int gameId, int playerId, InitalGamePack
             var connectionId = _playersByClientId[clientId].ConnectionId;
             await _hubContext.Clients.Client(connectionId).SendAsync(ClientHubMethods.InitializeGameState, initalGamePackage);
         }
-
+        public async Task NotifyReturnTribute(int gameId, int receiver, int payer, int cardsToBeReturned)
```

Comment:
I think `returnCardCount` might be a better name for `cardsToBeReturned`. `cardsToBeReturned` seems to suggest a list of actual cards that we are passing into this method.

Reply:
Done